src/classes/DRS_ContactMerge_CTRL.cls

```diff
@@ -0,0 +1,1036 @@
+    * @description Shows list of Contact records related to selected Duplicate Record Set
+    */
+    public PageReference showContactRelatedToDRS() {
+        //logic to display contacts related to selected DRS
+        Try {
+            step = 2;
+            this.contactResults = wrapSOQLResults(queryRecords());
+        } catch (exception ex) {
+            ApexPages.addMessage(new ApexPages.Message(ApexPages.Severity.Error, ex.getMessage()));
+        }
+        return null;
+    }
+
+    /******************************************************************************************************
+    * @description This will query the duplicate contacts related to the DRS which is picked
+    * @return the list of contacts
+    */
+    public list<SObject> queryRecords() {
+        contactResults.clear();
+        String whereString = new UTIL_Query()
+            .withFrom('DuplicateRecordItem')
+            .withSelectFields(driFieldNames)
+            .withWhere('DuplicateRecordSetId =:DRSrecordId')
+            .build();
+        whereString = 'Id IN ('+ whereString + ')';
+        String queryString = new UTIL_Query()
+            .withFrom('Contact')
+            .withSelectFields(listStrContactField)
+            .withWhere(whereString)
+            .build();
+        return Database.query(queryString);
+    }
+
+    /*******************************************************************************************************
+    * @description Wraps the SOQL results.
+    * @param contactResults The list of SObjects to wrap.
+    * @return List<ContactWrapper> Those records from the passed parameter that are of type Contact are returned
+    * in this list, inside wrappers.
+    */
+    public List<ContactWrapper> wrapSOQLResults(List<SObject> contactResults) {
+        List<ContactWrapper> res = new List<ContactWrapper>();
+        for (SObject returnedObject : contactResults) {
+            ContactWrapper thisResult = new ContactWrapper((Contact) returnedObject);
+            res.add(thisResult);
+        }
+        return res;
+    }
+    /*******************************************************************************************************
+    * @description Shows the next step of the wizard where user can see the diff of the records before merge.
+    */
+    public void nextStep() {
+
+        set<Id> mergeIds = new set<Id>();
+        for (ContactWrapper contactWrap : contactResults) {
+            if (contactWrap.selected) {
+                mergeIds.add(contactWrap.contactRecord.Id);
+            }
+        }
+        loadMergeCandidates(mergeIds);
+    }
+    /*******************************************************************************************************
+    * @description Shows the next step of the wizard where user can see the diff of the records before merge.
+    */
+    private void loadMergeCandidates(set<Id> mergeCandidates) {
```

Comment:
This method also looks identical to the variant in CON_ContactMerge_CTRL. This can't just be copy and pasted, it needs to be shared either through class inheritance or moving to a shared utility to service class.